packages/integration-sdk-cli/src/generator/template/src/validateInvocation.test.ts.hbs

```diff
@@ -0,0 +1,104 @@
+import {
```

Comment:
Should we consider getting rid of this or stubbing out something smaller? Perhaps comments instead of actual tests?